content/themis/installation/installation-from-sources.md

```diff
@@ -37,15 +37,6 @@ as it contains the necessary header files.
 
 In either case, we **strongly** recommend that you use the most recent version of these packages.
 
-{{< hint warning >}}
-**Note:**
-At the moment Themis is not fully compatible with OpenSSL 3.0.
```

Comment:
we should not just remove it, we should write smth like "Starting from Themis 0.15, it supports OpenSSL 3.0 as well as OpenSSL 1.1.1.." blahblahblah

Reply:
oh, yeah, good point